cli_tools/gce_ovf_import/README.md

```diff
@@ -1,7 +1,7 @@
 ## Compute Engine OVF Import
 
 The `gce_ovf_import` tool imports a virtual appliance in OVF format created in VMware environments
-to Google Compute Engine VM. It supports importing OVF and OVA archives.
+to Google Compute Engine VM or machine image. It supports importing OVF and OVA archives.
```

Comment:
I feel "machine image" is not clear in the context. I know it's GMI, but "machine image" sounds more general. Can we use GMI?

Reply:
Changed to "Google Compute Engine machine image". We shouldn't use acronyms. Most people external to Google will not know what a GMI is until it becomes widespread, like Amazon's AMIs.